HHS/Head-Start-TTADP
# [Prod] Add screenshot button to TTA hours widget, improve display on x-axis with large datasets

## Description of change
This PR adds two changes to the Total hours by TTA type widget on the regional dashboard
- There is a "save screenshot" button that creates and downloads an image of that graph to a user's computer
- The x-axis of the TTA hours graph is now dynamically adjusted based on how many x-axis ticks there might be

This PR also fixes a bug with a handful of reports where the "Goals & objectives" section would be blank for the approver
This PR also includes code to run a bulk indexing of activity report data into our elasticsearch.

## How to test
Use the TTA hours graph with a large dataset. See that fewer ticks are labelled based on how big the dataset is. Confirm that the save screenshot button works as well.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-509
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1245


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
